content/backend/README.md

```diff
@@ -18,7 +18,7 @@ cd dev/proxy
 ```sh
 cd src/Apps/LncdApp.Migrations
 # This is required for now, but it does not need to point to a real database
-export SqlServer__ConnectionString='Server=localhost,1433;Database=App;User Id=sa;Password=yourStrong(!)Password'
+export SqlServer__ConnectionString='Server=sql.local.lncd.pl,1433;Database=App;User Id=sa;Password=yourStrong(!)Password;Encrypt=false'
```

Comment:
If it

> does not need to point to a real database

then why change it? :thinking:

Reply:
Right, but I'd let it stay the proper way

Reply:
Okay. Does server name matter at all or will `localhost` work just as well? With `localhost` there will be no need for more changes if we ever decide to update our naming scheme again (and I prefer `mssql` instead of `sql` anyway :see_no_evil:).